indico/MaKaC/PDFinterface/conference.py

```diff
@@ -643,715 +632,742 @@ def _processTOCPage(self):
             c.setFont('Times-Bold', modifiedFontSize(30, self._fontsize))
-            height-=1*cm
-            c.drawCentredString(self._PAGE_WIDTH/2.0,height,\
-                    self._title)
-            self._drawWrappedString(c, "%s / %s"%(self._conf.getTitle(),self._title), width=inch, height=0.75*inch, font='Times-Roman', size=modifiedFontSize(9, self._fontsize), color=(0.5,0.5,0.5), align="left", maximumWidth=self._PAGE_WIDTH-3.5*inch, measurement=inch, lineSpacing=0.15)
-            c.drawRightString(self._PAGE_WIDTH-inch,0.75*inch,
-                    nowutc().strftime("%A %d %B %Y"))
+            height -= 1 * cm
+            c.drawCentredString(self._PAGE_WIDTH / 2.0, height, self._title)
+            self._drawWrappedString(c, "{} / {}".format(self._conf.getTitle(), self._title), width=inch,
+                                    height=0.75 * inch, font='Times-Roman', size=modifiedFontSize(9, self._fontsize),
+                                    color=(0.5, 0.5, 0.5), align="left", maximumWidth=self._PAGE_WIDTH - 3.5 * inch,
+                                    measurement=inch, lineSpacing=0.15)
+            c.drawRightString(self._PAGE_WIDTH - inch, 0.75 * inch, nowutc().strftime("%A %d %B %Y"))
             c.restoreState()
 
-    def laterPages(self,c,doc):
-
+    def laterPages(self, c, doc):
         c.saveState()
-        maxi=self._PAGE_WIDTH-2*cm
-        if doc.getCurrentPart().strip() != "":
-            maxi=self._PAGE_WIDTH-6*cm
-        self._drawWrappedString(c, "%s / %s"%(self._conf.getTitle(),self._title), width=1*cm, height=self._PAGE_HEIGHT-1*cm, font='Times-Roman', size=modifiedFontSize(9, self._fontsize), color=(0.5,0.5,0.5), align="left", lineSpacing=0.3, maximumWidth=maxi)
-        c.drawCentredString(self._PAGE_WIDTH/2.0,0.5*cm,i18nformat(""" _("Page") %d """)%(doc.page + self._firstPageNumber - 1))
-        c.drawRightString(self._PAGE_WIDTH-1*cm,self._PAGE_HEIGHT-1*cm,doc.getCurrentPart())
+        maxi = self._PAGE_WIDTH - 2 * cm
+        if doc.getCurrentPart().strip():
+            maxi = self._PAGE_WIDTH - 6 * cm
+        self._drawWrappedString(c, "{} / {}".format(self._conf.getTitle(), self._title), width=1 * cm,
+                                height=self._PAGE_HEIGHT - 1 * cm, font='Times-Roman',
+                                size=modifiedFontSize(9, self._fontsize), color=(0.5, 0.5, 0.5), align="left",
+                                lineSpacing=0.3, maximumWidth=maxi)
+        c.drawCentredString(self._PAGE_WIDTH / 2.0, 0.5 * cm,
+                            i18nformat(""" _("Page") {} """).format(doc.page + self._firstPageNumber - 1))
+        c.drawRightString(self._PAGE_WIDTH - 1 * cm, self._PAGE_HEIGHT - 1 * cm, doc.getCurrentPart())
         c.restoreState()
 
     def _defineStyles(self):
-        self._styles={}
+        self._styles = {}
 
-        dayStyle=getSampleStyleSheet()["Heading1"]
+        dayStyle = getSampleStyleSheet()["Heading1"]
```

Comment:
I'd add `stylesheet = getSampleStyleSheet()` at the beginning and replace it all over.